src/pages/BigCatNew.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react'
+
+BigCatNew extends Component {
```

Comment:
Make sure that you are declaring the class, otherwise, this will throw an error.
```suggestion
class BigCatNew extends Component {
```

Reply:
done